boot.php

```diff
@@ -10,18 +10,9 @@
 						unlink($this->getAssetsUrl('cache/redactor2_profiles.js'));
 					break;
 				}
-			});
-			
-			rex_extension::register('REX_FORM_DELETED', function (rex_extension_point $ep) {
-				$params = $ep->getParams();
-				$formParams = $params['form']->getParams();
-				
-				switch ($formParams['page']) {
-					case 'redactor2/profiles':
-						unlink($this->getAssetsUrl('cache/redactor2_profiles.js'));
-					break;
-				}
-			});
+			};
+			rex_extension::register('REX_FORM_SAVED', $deleteCacheFn);
```

Comment:
Eigentlich bräuchte das gar nicht in der boot.php sein, sondern würde in der page direkt selbst besser reinpassen